content/id/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -0,0 +1,234 @@
+
+### Mengamankan komunikasi
+
+Untuk mengonfigurasi etcd dengan _secure peer communication_, tentukan flag `--peer-key-file=peer.key` dan `--peer-cert-file=peer.cert`, dan gunakan https sebagai skema URL.
+
+Demikian pula, untuk mengkonfigurasi etcd dengan _secure client communication_, tentukan flag `--key-file=k8sclient.key` dan `--cert-file=k8sclient.cert`, dan gunakan https sebagai skema URL.
+
+### Membatasi akses klaster etcd
+
+Setelah konfigurasi komunikasi aman, batasi akses klaster etcd hanya ke server API Kubernetes. Gunakan otentikasi TLS untuk melakukannya.
+
+Sebagai contoh, utamakan pasangan kunci `k8sclient.key` dan `k8sclient.cert` yang dipercaya oleh CA `etcd.ca`. Ketika etcd dikonfigurasi dengan `--client-cert-auth` bersama dengan TLS, etcd memverifikasi sertifikat dari klien dengan menggunakan CA sistem atau CA yang dilewati oleh `--trusted-ca-file` flag. Menentukan flag `--client-cert-auth=true` dan `--trusted-ca-file=etcd.ca` akan membatasi akses ke klien dengan sertifikat `k8sclient.cert`.
+
+Setelah etcd dikonfigurasi dengan benar, hanya klien dengan sertifikat yang valid yang dapat mengaksesnya. Untuk memberikan akses server Kubernetes API, konfigurasikan dengan flag `--etcd-certfile=k8sclient.cert`,`--etcd-keyfile=k8sclient.key` dan `--etcd-cafile=ca.cert`.
+
+{{< note >}}
+otentikasi etcd saat ini tidak didukung oleh Kubernetes. Untuk informasi lebih lanjut, lihat masalah terkait [Mendukung Auth Dasar untuk Etcd v2](https://github.com/kubernetes/kubernetes/issues/23398).
+{{< /note >}}
+
+## Mengganti anggota etcd yang gagal
+
+etcd klaster mencapai ketersediaan tinggi dengan mentolerir kegagalan kecil anggota. Namun, untuk meningkatkan kesehatan keseluruhan klaster, ganti anggota yang gagal segera. Ketika banyak anggota gagal, gantilah satu per satu. Mengganti anggota yang gagal melibatkan dua langkah: menghapus anggota yang gagal dan menambahkan anggota baru.
+
+Meskipun etcd menyimpan ID anggota unik secara internal, disarankan untuk menggunakan nama unik untuk setiap anggota untuk menghindari kesalahan manusia. Sebagai contoh, sebuah klaster etcd three-member. Jadikan URL-nya, member1=http://10.0.0.1, member2=http://10.0.0.2, and member3=http://10.0.0.3. Ketika member1 gagal, ganti dengan member4=http://10.0.0.4.
+
+1. Dapatkan ID anggota yang gagal dari member1:
+
+    `etcdctl --endpoints=http://10.0.0.2,http://10.0.0.3 member list`
+
+      The following message is displayed:
```

Comment:
Hello, @ariscahyadi thank you for reviewing my PR.
Sorry beforehand, when I first glance at this sentence. I thought this sentence is part of the output from `etcdctl --endpoints=http://10.0.0.2,http://10.0.0.3 member list` command because of its indentation.
I will provide a translation for this also in my next commit.